.travis.yml

```diff
@@ -5,14 +5,15 @@ before_install:
   - gem update --system --no-doc
   - gem install bundler --no-doc
   - gem update bundler --no-doc
-  - smoke/provision.sh
+  - bin/provision.sh
 rvm:
   - 2.2.8
   - 2.3.5
   - 2.4.2
   - ruby-head
 env:
   - TEST_TASK=spec
+  - influx_version=1.4.2
```

Comment:
This line is not necessary, there are a bunch of `TEST_TASK=smoke` entries in the `matrix.include` section below, which also define the `influx_version`.

Reply:
I was trying to resolve this error on the Travis tests 

Failure/Error: raise InfluxDB::ConnectionError, "Tried #{retry_count - 1} times to reconnect but failed

It didn't resolve that anyway, so I'll take it out and keep working on it.